lib/client-options.js

```diff
@@ -102,9 +102,6 @@ function extend(baseOptions, userOptions) {
         'the following values are valid contact points: ipAddress, hostName or ipAddress:port', i, hostName));
     }
   }
-  if (!options.logEmitter) {
```

Comment:
I noticed that when using `extends` logging stopped working because of this.  Felt it'd be better instead to not do this so the default behavior is maintained when the user doesn't provide a `logEmitter`